coreml/export.py

```diff
@@ -247,7 +247,7 @@ def export_image_encoder(
     mlmodel = ct.convert(
         traced_model,
         inputs=[
-            ct.TensorType(name="image", shape=(1, 3, 1024, 1024)),
+            ct.ImageType(name="image", shape=(1, 3, 1024, 1024), scale=1/255.0, bias=[0, 0, 0])
```

Comment:
Not sure we can just do this and have it work, let me see if the output still matches torch